src/plugins/cloudinary/common.js

```diff
@@ -102,109 +102,13 @@ const cloudinaryErrorsConverter = ({ errorMsg, publicId, cloudName, statusCode }
   return error;
 };
 
-const codecShorthandTrans = short => {
```

Comment:
We decided not to test for specific codec support anymore, since almost all browsers now support them all.
With the default `f_auto` there shouldn't be an issue ever.
There are some edge cases in which old Safari's for example might try playing an AV1 and will have to fall-back to the next `sourceType`, which will cause a console error but nothing fatal. And if that's a concern we leace it to the implementor to prevent.